ui/src/components/VolumeCharts.js

```diff
@@ -30,15 +30,17 @@ export const VolumeThroughputChart = ({
   volumeName: string,
 }) => {
   const { series, startingTimeStamp, isLoading } = useSymetricalChartSeries({
-    getQueryAbove: (timeSpanProps: TimeSpanProps) =>
+    getAboveQueries: (timeSpanProps: TimeSpanProps) => [
       getVolumeThroughputWriteQuery(instanceIp, deviceName, timeSpanProps),
-    getQueryBelow: (timeSpanProps: TimeSpanProps) =>
+    ],
+    getBelowQueries: (timeSpanProps: TimeSpanProps) => [
       getVolumeThroughputReadQuery(instanceIp, deviceName, timeSpanProps),
+    ],
     transformPrometheusDataToSeries: useCallback(
       (prometheusResultAbove, prometheusResultBelow) =>
```

Comment:
`([prometheusResultAbove], [prometheusResultBelow]) =>` that would avoid `[0]` accessors below